lib/rex/proto/secauthz/well_known_sids.rb

```diff
@@ -0,0 +1,144 @@
+module Rex::Proto::Secauthz
+  module WellKnownSids
+
+    # https://learn.microsoft.com/en-us/windows/win32/secauthz/well-known-sids
+
+    SECURITY_NULL_SID_AUTHORITY = 'S-1-0'.freeze
+    SECURITY_WORLD_SID_AUTHORITY = 'S-1-1'.freeze
+    SECURITY_LOCAL_SID_AUTHORITY = 'S-1-2'.freeze
+    SECURITY_CREATOR_SID_AUTHORITY = 'S-1-3'.freeze
+    SECURITY_NT_AUTHORITY = 'S-1-5'.freeze
+
+    SECURITY_NULL_RID = 0
+    SECURITY_WORLD_RID = 0
+    SECURITY_LOCAL_RID = 0
+    SECURITY_LOCAL_LOGON_RID = 1
+    SECURITY_CREATOR_OWNER_RID = 0
+    SECURITY_CREATOR_GROUP_RID = 1
+
+    SECURITY_CREATOR_OWNER_SID = "#{SECURITY_CREATOR_SID_AUTHORITY}-#{SECURITY_CREATOR_OWNER_RID}".freeze
```

Comment:
I believe we should be able to add `# frozen_string_literal: true` at the top of the file without the need for the individual `.freeze` calls